packages/e2e/src/tools/multi-delegation-data-gen/example.json

```
@@ -0,0 +1,31 @@
+{
+  "startingFunds": 1000000000000,
+  "stakeDistribution": [
+    0.2,
+    0.2,
+    0.6
+  ],
+  "outputs": {
+    "amount": {
+      "min": 1000000000,
+      "max": 1000000000
+    },
+    "count": {
+      "min": 1,
+      "max": 1
+    },
+    "period": 1
+  },
+  "inputs": {
```

This got me confused a little bit, how about:
```suggestion
  "utxoIn": {
    "amount": {
      "min": 1000000000,
      "max": 1000000000
    },
    "count": {
      "min": 1,
      "max": 1
    },
    "period": 1
  },
  "utxoOut": {
```